libs/wingsdk/src/target-awscdk/bucket.ts

```diff
@@ -55,6 +56,15 @@ export class Bucket extends cloud.Bucket {
     });
   }
 
+  public addFile(fileName: string, path: string): void {
+    path = isAbsolute(path)
+      ? path
+      : resolve(process.env.WING_SOURCE_DIR ?? "", path);
```

Comment:
Nice! 
(but it probably fail if a few files are running on the same process, as it will be overridden, unfortunatly) 
can you please change it to `App.of(scope).entrypointDir`
(like in here: https://github.com/winglang/wing/blob/86779dbddea789f84496437fce03d1fabd739144/libs/wingsdk/src/cloud/website.ts#L62-L65)
Since this argument is more consistent

Reply:
Sure.